ApplicationLibCode/ProjectDataModel/RimDepthTrackPlot.cpp

```diff
@@ -1199,10 +1202,17 @@ void RimDepthTrackPlot::initAfterRead()
         m_nameConfig->setCustomName( m_plotWindowTitle );
     }
 
-    if ( RimProject::current()->isProjectFileVersionEqualOrOlderThan( "2022.06.2" ) &&
-         !m_nameConfig->customName().isEmpty() )
+    if ( RimProject::current()->isProjectFileVersionEqualOrOlderThan( "2022.06.2" ) )
     {
-        m_namingMethod = RiaDefines::ObjectNamingMethod::CUSTOM;
+        if ( !m_nameConfig->customName().isEmpty() )
+        {
+            m_namingMethod = RiaDefines::ObjectNamingMethod::CUSTOM;
+        }
+
+        if ( m_depthOrientation_OBSOLETE == DepthOrientation_OBSOLETE::HORIZONTAL )
+            m_depthOrientation = RiaDefines::Orientation::HORIZONTAL;
+        else
+            m_depthOrientation = RiaDefines::Orientation::VERTICAL;
```

Comment:
Do you need to do backward compatibility when the enum has same name? The xml should be either "VERTICAL" or "HORIZONTAL"?